pkgs/tools/admin/bigeye-cli/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, python3
+, fetchPypi
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "bigeye_cli";
```

Comment:
```suggestion
  pname = "bigeye-cli";
```

Reply:
On [pypi it has underscore](https://pypi.org/project/bigeye-cli/#files), I get an error if I do this:

```
trying https://files.pythonhosted.org/packages/source/b/bigeye-cli/bigeye-cli-0.3.42.tar.gz
...
curl: (22) The requested URL returned error: 404
trying https://pypi.io/packages/source/b/bigeye-cli/bigeye-cli-0.3.42.tar.gz
curl: (22) The requested URL returned error: 404
```